datamodel/initdb.d/05_reference_implementation_support.sql

```diff
@@ -108,6 +132,14 @@ UNION
     NULL::text AS empty_indicator_code,
     NULL::text AS document_id,
     NULL::text AS reason,
+    NULL::text as operations_event_type_code,
+    NULL::text as publisher_role,
+    NULL::text as publisher_code_list_provider,
+    NULL::text as event_location,
+    NULL::text as port_call_service_type_code,
+    NULL::text as facility_type_code,
+    NULL::text as vessel_position,
+    NULL::text as publisher,

```

Comment:
This is wrong? All these fields should be populated from the operations_event...?

Reply:
You are populating the view with Null values...